docs/runtimes/configuring-job-resources.md

```diff
@@ -392,14 +409,16 @@ a merge with precedence is performed with the project and MLRun config levels.
 the runtime node selector is used. If a specific `executor_node_selector` is defined, it takes precedence. 
 A merge with precedence is then performed with the project and MLRun config levels.
 
-This logic becomes part of the Spark CRD, ensuring that it is consistently applied during the job execution. 
+
 
 ### Best Practice
 
 Node selection is often used for assigning jobs/pods to GPU nodes. But not all jobs/pods benefit from a GPU node.
-For example, a Databricks “helper” pod runs in a Spark service on Databricks and doesn’t follow the node-selector 
-(and doesn't benefit from being assigned to a GPU node). 
-A Spark Function includes an executor and a driver; the driver also does not benefit from a GPU node.
+For example:
+- With Databricks, the node selector is only relevant for the "helper" pod running in the MLRun Kubernetes cluster,  
+and it behaves similarly to how node selectors are applied in Kubejob. It does not affect the actual Databricks cluster, 
+which is not based on Kubernetes and does not run in our cluster: node selectors have no significance in that context.
```

Comment:
I don't think that's true. I would remove it
```suggestion
which does not run in our cluster: node selectors have no significance in that context.
```

Reply:
It is true, but I'm not sure it's relevant